release-notes/unreleased/1715.feat.rst

```diff
@@ -0,0 +1,26 @@
+New opt-in feature to support fractioanl gates is added to :class:`.IBMBackend`.
+IBM backends currently support dynamic circuits and fractional gates exclusively and
+two features cannot be used in the same primitive job.
+Since Qiskit Target model doesn't represent such constraint,
+we adopted the opt-in approach, where your backend target includes only 
+fractional gates (control flow instructions) when the backend is (not) opted.
+This feature is controlled when you retrieve the target backend from the :class:`.QiskitRuntimeService`.
+
+.. code-block:: python
+
+    from qiskit_ibm_runtime import QiskitRuntimeService
+
+    backend = QiskitRuntimeService(channel="ibm_quantum").backends(
+        "name_of_your_backend", 
+        use_fractional_gates=True,
+    )[0]
+
+When the fractional feature is enabled, transpiled circuits may have
+shorter depth compared with the conventional IBM basis gates, e.g. [sx, rz, ecr].
+
+When you use control flow instructions, e.g. ``if_else``, in your circuit,
+you must disable the fractional gate feature to get executable ISA circuits.
+The choice of the instruction set is now responsibility of users.
```

Comment:
We should also point out that PEA and PEC are also not supported with fractional gates, and that people should use `use_fractional_gates=False` if they plan to use those error mitigation methods in the primitives.